lib/rets/cookie_manager.rb

```diff
@@ -0,0 +1,14 @@
+require 'httpclient'
+require 'http-cookie'
+
+module Rets
+  class CookieManager < WebAgent::CookieManager
+    attr_reader :jar
+
+    def initialize(cookies_file = nil, format = HTTPClient::WebAgentSaver, jar = HTTP::CookieJar.new)
+      super(cookies_file, format)
+      @jar = jar
+      load_cookies if @cookies_file
+    end
+  end
+end
```

Comment:
This whole class is a stop-gap for https://github.com/nahi/httpclient/pull/254.

I was a bit loathe to write our own cookie manager just to inject and expose the cookie jar (I'm a bit confused about what parts of HTTPClient rely on what cookie implementations as it seems like internally it uses WebAgent::Cookie some places).